src/components/logo/logo.tsx

```diff
@@ -27,14 +30,60 @@ const mapStateToProps = state => ({
  */
 @connect(mapStateToProps)
 @withPlayer
+@withEventManager
 @withLogger(COMPONENT_NAME)
 @withText({logoText: 'controls.logo'})
 class Logo extends Component<any, any> {
+  private _logoRef: RefObject<HTMLAnchorElement> = createRef<HTMLAnchorElement>();
+
+  /**
+   * before component mounted, set initial state of the logo
+   *
+   * @returns {void}
+   * @memberof Logo
+   */
+  public componentWillMount(): void {
```

Comment:
This lifecycle is deprecated, and should not be used for state initialisation 
See here:
https://legacy.reactjs.org/docs/react-component.html#unsafe_componentwillmount